src/test/java/uk/ac/sanger/sccp/stan/EntityFactory.java

```diff
@@ -190,6 +191,19 @@ public static Tissue makeTissue(Donor donor, SpatialLocation sl) {
         return new Tissue(id, "TISSUE "+id, String.valueOf(id%7), sl, donor, getMedium(), getFixative(), getHmdmc());
     }
 
+    public static ReagentPlate makeReagentPlate(String barcode) {
+        int plateId = ++idCounter;
+        ReagentPlate rp = new ReagentPlate(barcode);
+        rp.setId(plateId);
+        ReagentPlateType rpType = rp.getPlateType();
+        int[] slotId = {100*plateId};
```

Comment:
What is this exactly doing and why we are using 100*plateId?

Reply:
It is creating a reagent plate for the test, with the appropriate slots. If the plate id is (for instance) 56, then the slots will have ids 5600 to 5695. I do this so that if I am debugging a test, it is easy to see what slot ids are associated with what plate id.